src/reanimated2/threads.ts

```diff
@@ -130,6 +130,11 @@ export function runOnUI<Args extends unknown[], ReturnValue>(
   };
 }
 
+// @ts-expect-error Check `executeOnUIRuntimeSync` overload above.
+export function executeOnUIRuntimeSync<Args extends unknown[], ReturnValue>(
+  worklet: (...args: Args) => ReturnValue
+): WorkletFunction<Args, ReturnValue>;
```

Comment:
It doesn't return a worklet actually.

```suggestion
export function executeOnUIRuntimeSync<Args extends unknown[], ReturnValue>(
  worklet: (...args: Args) => ReturnValue
): (...args: Args) => ReturnValue;
```